microscopium/preprocess.py

```diff
@@ -171,7 +173,7 @@ def maxes(fns):
     return maxes
 
 
-def stretchlim(im, bottom=0.01, top=None, mask=None):
+def stretchlim(im, bottom=0.01, top=None, mask=None, in_place=False):
```

Comment:
Maybe the default should be 0.001 here? You recommended this, and I had success using this value with the Cellomics datasets.